src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs

```diff
@@ -77,8 +77,8 @@ private void Rec_PanUpdated(object sender, PanUpdatedEventArgs e)
             }
 
             var currentPos = m_startSlideLocation - e.TotalX;
-            m_lastId = currentId;
-            var index = Math.Max(Config.MinValue - 0.45, Math.Min(Config.MaxValue + 0.45, CalculateIndex(currentPos)));
+            m_lastId = currentId;
```

Comment:
Add an if to the top of this method and return if Config is null. Should not continue with NULL config. As other bugs might occur.